bootloader/lkenv/lkenv.go

```diff
@@ -232,13 +236,41 @@ func (l *Env) Load() error {
 	return nil
 }
 
+type compatErrNotExist struct {
+	err error
+}
+
+func (e compatErrNotExist) Error() string {
+	return e.err.Error()
+}
+
+func (e compatErrNotExist) Unwrap() error {
+	// for go 1.9 xerrors compatibility
+	go1_9Regex := regexp.MustCompile(`go1\.9(\.[0-9]+){0,1}$`)
+	if go1_9Regex.MatchString(runtime.Version()) {
```

Comment:
no longer hard-coding which version of Go, we just inspect the error to see if it can do Unwrap, and if so then use it, if not return os.ErrNotExist directly